lib/moonshot/build_mechanism/github_release.rb

```diff
@@ -159,7 +163,9 @@ def validate_commit
       sh_step(cmd, msg: "Validate commit #{@sha}.") do |_, out|
         @commit_detail = out
       end
-      @ci_statuses = check_ci_status(@sha)
+      @ci_statuses = check_ci_status(@sha) unless 
```

Comment:
I think you want to do this `if` the value is false, not `unless` the value is false, right?